src/main/java/models/Song.java

```diff
@@ -0,0 +1,110 @@
+package models;
+
+import static enumeration.Genre.*;
+
+import enumeration.Genre;
+
+/**
+ * This class represent song, which will be executed by the audio player
+ */
+public class Song {
+    private String title;
+    private Genre genre;
+    private Singer singer;
+    private int timing;
+    private static final String DEFAULT_TITLE = "Unknown";
+    private static final int DEFAULT_TIMING = 1;
+
+    /**
+     * The constructor initialize the class fields with the values from the parameters if they are valid, otherwise it
+     * initialize the fields with default values
+     *
+     * @param title  - this parameter is the title of the song
+     * @param singer - this parameter is the singer of the song
+     * @param genre  - this parameter is the genre of the song
+     * @param timing - this parameter is timing of the song
+     */
+    public Song(String title, Singer singer, Genre genre, int timing) {
+        this.title = getValidTitle(title);
+        this.genre = getValidGenre(genre);
+        this.timing = getValidTiming(timing);
+        this.singer = getValidSinger(singer);
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public Singer getSinger() {
+        return singer;
+    }
+
+    public int getTiming() {
+        return timing;
+    }
+
+    /**
+     * This method check if the song's singer is valid. If it is the singer is returned, otherwise the default value is returned
+     * from Singer class
+     *
+     * @param singer - this parameter is the singer of the song
+     * @return - this method return the singer if the parameter is valid, otherwise return new singer, which will be populated with default values
+     */
+    private Singer getValidSinger(Singer singer) {
+        return singer != null ? singer : new Singer(null);
```

Comment:
new Singer(null);
This could be changed in `new Singer()` and in `Singer` class to chain constructors.
It's not good practice to pass `null`s when initialise Objects